app/src/main/java/org/wordcamp/android/adapters/SessionsListAdapter.java

```diff
@@ -52,10 +55,16 @@ public View getView(final int position, View convertView, ViewGroup parent) {
         holder.title.setText(Html.fromHtml(sessionDB.getTitle()));
 
         if (sessionDB.getLocation() == null || sessionDB.getLocation().isEmpty()) {
-            holder.location.setVisibility(View.GONE);
+            holder.locationContainer.setVisibility(View.GONE);
         } else {
-            holder.location.setVisibility(View.VISIBLE);
+            holder.locationContainer.setVisibility(View.VISIBLE);
             holder.location.setText(Html.fromHtml(sessionDB.getLocation()));
+            if (speakersForSession.containsKey(sessionDB.getPost_id())) {
+                holder.speakers.setVisibility(View.VISIBLE);
+                holder.speakers.setText(" – " + speakersForSession.get(sessionDB.getPost_id()));
```

Comment:
Use `R.string.date_separator` instead of `" – "`, and rename `date_separator` to `separator`